server/src/main/java/org/opensearch/index/translog/RemoteFsTranslog.java

```diff
@@ -522,23 +525,26 @@ private class RemoteFsTranslogTransferListener implements TranslogTransferListen
         /**
          * Generation for the translog
          */
-        private final Long generation;
+        private final long generation;
 
         /**
          * Primary Term for the translog
          */
-        private final Long primaryTerm;
+        private final long primaryTerm;
 
-        RemoteFsTranslogTransferListener(Long generation, Long primaryTerm) {
+        private final long maxSeqNo;
+
+        RemoteFsTranslogTransferListener(long generation, long primaryTerm, long maxSeqNo) {
             this.generation = generation;
             this.primaryTerm = primaryTerm;
+            this.maxSeqNo = maxSeqNo;
         }
 
         @Override
         public void onUploadComplete(TransferSnapshot transferSnapshot) throws IOException {
             maxRemoteTranslogGenerationUploaded = generation;
             minRemoteGenReferenced = getMinFileGeneration();
-            logger.trace("uploaded translog for {} {} ", primaryTerm, generation);
+            logger.info("Successfully uploaded translog for primary term = {}, generation = {}, maxSeqNo = {}", primaryTerm, generation, maxSeqNo);
```

Comment:
This would emit logs for all translog uploads and can be noisy. Shall we change the log level to TRACE?